### ISSUE TITLE
User Dashboard: Neighborhood Completion Graph

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#413]
For the user dashboard (visible to all registered users), we should include a neighborhood completion graph that nicely shows how much they've contributed to the neighborhoods they've audited in %.

**Comment by misaugstad** [comment]
I totally agree with this. As someone who is super susceptible to gamification, this is the number 1 thing I want. Every time do a mission or two, I go to the demo page to see that the neighbourhood percentage has gone up a bit!
